src/overlay/components/visual/ViewStats.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import './ViewStats.scss';
+import viewersUp from '../../img/Header/ViewersUp.svg';
+import viewersDown from '../../img/Header/ViewersDown.svg';
+import viewCount from '../../img/Header/ViewCount.svg';
+import {ViewersState} from "../../../reducers/viewers";
+import { connect } from "react-redux";
+import { ApplicationState } from "../../../reducers";
+
+class ViewStats extends React.Component<ViewersState> {
+    render() {
+        return <div className="viewers">
+            <span className="viewer-count">
+                <img src={viewCount} alt="View Count"/>
+                <span>{this.props.viewers}</span>
+            </span>
+            <span className="viewer-change">
+                <img className="change" src={(this.props.change >= 0 ? viewersUp : viewersDown)} alt="Viewer Change"/>
```

Comment:
Rather than an explicit image here, the class .viewer-change could add the glyph as a :before pseudoelement. Keeps the markup cleaner. We could also use another class to tell it whether the arrow should point up or down. If we use the same arrow and just rotate it, we could even transition between the two.